assets/sass/components/_prisoner-profile.scss

```diff
@@ -43,3 +43,39 @@
     background-color: govuk-colour("purple");
   }
 }
+
+.prisoner-profile-visits {
+  & td:nth-child(1) {
+    width: 15%;
```

Comment:
Using separate tables for each month's visits, so having to fix widths otherwise each table ends up different.